qa/TL0_multigpu/test_nofw.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
 # used pip packages
-pip_packages='${python_test_runner_package} numpy librosa==0.8.1 scipy nvidia-ml-py==11.450.51 psutil dill cloudpickle opencv-python'
+pip_packages='${python_test_runner_package} numpy librosa==0.8.1 scipy nvidia-ml-py==11.450.51 psutil dill cloudpickle opencv-python pillow'
```

Comment:
so that we can save images on failures (other jobs have pillow, those two were missing)